examples/sample-allOption.yml

```diff
@@ -0,0 +1,71 @@
+id: myapp
+name: MyApp Service (powered by WinSW)
+description: This service is a service created from a sample configuration
+executable: java
+#serviceaccount:
+#    domain: YOURDOMAIN
+#    user: useraccount
+#    password: Pa55w0rd
+#    allowservicelogon: yes
+#onFailure:
+#    -
+#        action: restart
+#        delay: 10 sec
+#    -
+#        action: restart
+#        delay: 20 sec
+#    -
+#        action: reboot
+#resetFailureAfter: 01:00:00
+#securityDescriptor: security descriptor string
+#arguments: >
+#    -classpath 
+#    c:\cygwin\home\kohsuke\ws\hello-world\out\production\hello-world 
+#    test.Main
+#startArguments: start arguments
+#workingdirectory: C:\myApp\work
+priority: Normal
+stopTimeout: 15 sec
+stopParentProcessFirst: true 
+#stopExecutable: '%BASE%\stop.exe'
+#stopArguments: -stop true
+startMode: Automatic
+#delayedAutoStart: true
+#serviceDependencies:
+#    - Eventlog
+#    - W32Time
+waitHint: 15 sec
+sleepTime: 1 sec
+#interactive: true
+log:
+#    logpath: '%BASE%\logs'
+    mode: append
+#env:
+#    MY_TOOL_HOME: 'C:\etc\tools\myTool'
```

Comment:
- Shall we support a similar syntax for XML in v3?
- Shall we support `name:` and `value:` pairs?

Reply:
Sure. I am okay with that! I will update this.

Reply:
> * Shall we support a similar syntax for XML in v3

Sorry, I did not get this???

Reply:
I believe @NextTurn asks whether we should change XML syntax in v3 so that YAML and XML are aligned

Reply:
What would be good?? I am open for both. However I already change YAML config into XML syntax. I will push that. But still I can roll back to previous syntax if you wish.